src/software/world/team.h

```diff
@@ -162,6 +162,25 @@ class Team
      */
     std::vector<Robot> getAllRobotsExceptGoalie() const;
 
+    /**
+     * Given a team, finds the robot on that team that is closest to a reference point.
```

Comment:
Nit:

I would simplify this and say 'Finds the robot on a team that is closest to the reference point'